README.md

```diff
@@ -1,5 +1,5 @@
-# front-end
-A UI for interacting with the data stored in the PDC service
+# Philanthropy Data Commons Front-End
```

Comment:
Reminder: This appears only in the README here in GitHub (and when developers clone the project). It describes the code repository, not the deployed website.

Reply:
<img width="582" alt="image" src="https://github.com/PhilanthropyDataCommons/front-end/assets/208884/e3b7fb97-1d2d-4ef1-8686-bf3d0e0b9a42">


Apparently "Front-end" is for adjective (e.g. front-end developer), whereas "Front End" is for a noun

Reply:
Ah, that's some clean guidance. Thanks!

Reply:
@slifty I've made this change but am leaving this unresolved, since I still want other reviewers to see my original comment.